### ISSUE TITLE
what is the result or output that we want in the autoencoder example?

### ISSUE THREAD

**Issue by guotong1988** [nlintz/TensorFlow-Tutorials#51]
https://github.com/nlintz/TensorFlow-Tutorials/blob/master/06_autoencoder.py
I don't know the goal of the program..

**Comment by awoerner92** [comment]
The program is training an autoencoder to reconstruct the images from the MNIST dataset. The input are the images to which random noise (following a binomial distribution) is added in order to make the model more robust. The output is the reconstruction of the corrupted input. In the end, the goal is to reduce the cost, i.e. the difference between original input and the reconstruction.
As console output you get the cost on the test set for each epoch.

**Comment by nlintz** [comment]
Thanks for the awesome reply @awoerner92, you are 100% correct :-)
